WordPress/gutenberg
# Remove rename rn UI tests

## Description
This PR removes 1 React Native E2E UI tests for Gutenberg Mobile that is covered on other tests: `should be able to add a new Paragraph block`

This PR also includes renaming changes, I initially thought the 3 features were the same and was ready to combine them but realized there are 3 different blocks with typos, fixed the typos in this PR.

## Testing Instructions
Ensure that React Native UI Tests still pass on CI. PR on gutenberg-mobile: https://github.com/wordpress-mobile/gutenberg-mobile/pull/4608

## Screenshots 
N/A

## Types of changes
Removed redundant tests and fixed typos

## Checklist:
